style/about.css

```diff
@@ -0,0 +1,147 @@
+body{
+    background-color: black;
+    width: auto; 
+    min-height:100vh;
+    height: 100vh;
+    margin:0;
+    font-family: helvetica;
+    display:grid;
+    flex-direction: column;
+    font-weight: 300;
+    grid-template-columns: .1fr 1fr 1fr 1fr 1fr 1fr .2fr;
+    grid-template-rows: .4 auto .2%;
+    color:#ebecd1; ;
+    justify-content:space-evenly;
```

Comment:
We should make sure our CSS follows consistent formatting. VSCode can help by using the "Format Document" option in the command palette.